theme/assets/stylesheets/cleura-palette.css

```diff
@@ -0,0 +1,4 @@
+[data-md-color-scheme="cleura"] {
+    --md-primary-fg-color: #FF6600;
```

Comment:
I didn't double check the values here. But the rest does make sense. Thanks for adding a palette scheme.